Source/LinqToDB/Common/Configuration.cs

```diff
@@ -40,6 +41,22 @@ public static class Configuration
 		/// queries, so this optimization could be used for <see cref="CommandBehavior.Default"/> too.
 		/// </summary>
 		public static bool OptimizeForSequentialAccess = false;
+		
+		/// <summary>
+		/// Determines the length after which logging of binary data in SQL will be truncated.
+		/// Use a value of -1 to disable. Set to 0 to truncate all binary data.
+		/// This is to avoid Out-Of-Memory exceptions when getting SqlText from <see cref="TraceInfo"/>
+		/// or <see cref="IExpressionQuery"/> for logging or other purposes.
+		/// </summary>
+		public static int MaxBinaryParameterLengthLogging { get; set; } = 100;
+
+		/// <summary>
+		/// Determines the length after which logging of string data in SQL will be truncated.
+		/// Use a value of -1 to disable. Set to 0 to truncate all binary data.
+		/// This is to avoid Out-Of-Memory exceptions when getting SqlText from <see cref="TraceInfo"/>
+		/// or <see cref="IExpressionQuery"/> for logging or other purposes.
+		/// </summary>
+		public static int MaxStringParameterLengthLogging { get; set; } = 200;
```

Comment:
I believe that commentary about specific values belongs in the `<remarks />` tag instead of in the `<summary />` tag. @MaceWindu can correct me if he doesn't like this?
```suggestion
		/// <summary>
		/// Determines the length after which logging of string data in SQL will be truncated.
		/// Truncation is recomended to avoid Out-Of-Memory exceptions when getting SqlText from <see cref="TraceInfo"/>
		/// or <see cref="IExpressionQuery"/> for logging or other purposes.
		/// </summary>
		/// <remarks>
		///  This value defaults to 200. 
		/// Use a value of -1 to disable truncation.
		/// Use a value of 0 to truncate all data.
		/// </remarks>
		public static int MaxStringParameterLengthLogging { get; set; } = 200;
```

Reply:
I'll defer to you and @MaceWindu on that one.